types/rot-js/index.d.ts

```diff
@@ -353,7 +353,7 @@ export interface RNGable {
     getUniformInt(lowerBound: number, upperBound: number): number;
     getNormal(mean?: number, stddev?: number): number;
     getPercentage(): number;
-    getWeightedValue<T>(data: { T: number }): T;
+    getWeightedValue<T>(data: { T: number }): T; // tslint:disable-line no-unnecessary-generics (TODO: This is probably wrong)
```

Comment:
Looks like the preferred solution here is to just use `any`, as much as I instinctively flinch when I do so.  Something like:

```
getWeightedValue(data: {any: number}): any;
```

I'll approve as-is but will also approve changes along these lines :)

Reply:
The problem is that the identifier on the left of the colon is a property name, not the name of a type. How is this function expected to be called? Using the above example you would need to call it like `getWeightedValue({ any: 42 })` which clearly isn't right.

Reply:
It's possible for the key to be a symbol as well; I suggested 'any' to coincide with the docs:

`* @param {object} data key=whatever`

Though `string | Symbol` would probably work as well